docs/user_guide/datasets.rst

```diff
@@ -0,0 +1,332 @@
+Datasets
+==========
+
+Revisiting the Boston Housing Dataset
+-------------------------------------
+
+Introduction
+^^^^^^^^^^^^^^^^^
+
+The Boston Housing dataset is one of the datasets currently callable in 
+:mod:`fairlearn.datasets` module. In the past, it has commonly been used for 
+benchmarking in popular machine learning libraries, including scikit-learn and 
+open-ml. However, as the machine learning community has developed awareness 
+about fairness issues with popular benchmarking datasets, the Boston Housing 
+data has been phased out of many libraries. We migrated the dataset to  
+fairlearn after it was phased out of scikit-learn in June 2020. The dataset  
+remains in fairlearn as an example of how system racism can occur in data  
+and to show the effect of fairlearn's assessment and pre-/in-/post-processing  
+tools on real, problematic data. 
+
+**We wrote this blog post to achieve the following goals:**
+  * Educate users about the history of the dataset and its bias
+  * Show users how bias manifests in the data and in downstream modelling tasks
+  * Suggest best practices for dealing with the Boston Housing data and 
+  alternative benchmarking datasets
+
+.. _dataset_origin:
+
+Dataset Origin and Use
+^^^^^^^^^^^^^^^^^^^^^^
+
+Contrary to expectation, the Boston Housing dataset was not developed for 
+economic purposes. `Harrison and Rubenfield (1978) <https://deepblue.lib.umich.edu/bitstream/handle/2027.42/22636/0000186.pdf?sequence=1&isAllowed=y>`_ 
+developed the dataset to illustrate the issues with using housing market data 
+to measure consumer willingness to pay for clean air. The authors use a 
+`hedonic pricing <https://www.investopedia.com/terms/h/hedonicpricing.asp>`_ 
+approach, which assumes that the price of a good or service can be modeled as a 
+function of features both internal and external to the good or service. The 
+input to this model was a dataset comprising the Boston Standard Metropolitan 
+Statistical Area, with the nitric oxides concentration (*NOX*) 
+serving as a proxy for air quality. 
+The paper sought to estimate the median value of owner-occupied homes (now 
+*MEDV*), and included the remaining variables to measure other neighborhood 
+characteristics. Further, the authors took the derivative of their housing 
+value equation with respect to nitric oxides concentration 
+to measure the "amount of money households
+were willing to pay with respect to air polution levels in their census 
```

Comment:
*pollution*

Reply:
good catch, thanks! will be resolved in next commit.